### PR TITLE
BETA: Register hymoonz.is-a.dev

### PR DESCRIPTION
**Pull request by hymoonz**
Added `hymoonz.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/hymoonz.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "hymoonz",
+        "email": "bilal190620@gmail.com"
+    },
+    "record": {
+        "CNAME": "hymoonz.github.io"
```
Comment (DEV-DIBSTER) [inline domains/hymoonz.json:7]: 404 CNAME.

File: domains/moonzdevelopment.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "hymoonz",
+        "email": "bilal190620@gmail.com"
+    },
+    "record": {
+        "TXT": "replit-verify=8cf3fb31-5df3-4217-b863-7d4088af08ec"
```
Comment (DEV-DIBSTER) [inline domains/moonzdevelopment.json:7]: TXT record alone doesn't do anything.

File: domains/moonzdev.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "hymoonz",
+        "email": "bilal190620@gmail.com"
+    },
+    "record": {
+        "CNAME": "8cf3fb31-5df3-4217-b863-7d4088af08ec.id.repl.co"
```
Comment (DEV-DIBSTER) [inline domains/moonzdev.json:7]: No content displayed.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/hymoonz.json (hunk 1 above)
Comment (hymoonz) [reply domains/hymoonz.json:7]: done sir

File: domains/moonzdevelopment.json (hunk 2 above)
Comment (hymoonz) [reply domains/moonzdevelopment.json:7]: done sir

File: domains/hymoonz.json (hunk 1 above)
Comment (DEV-DIBSTER) [reply domains/hymoonz.json:7]: Redirect loop now.

File: domains/moonzdevelopment.json (hunk 2 above)
Comment (DEV-DIBSTER) [reply domains/moonzdevelopment.json:7]: No changes were made to the file though...?

Comment (hymoonz) [reply domains/moonzdevelopment.json:7]: No sir

Comment (hymoonz) [reply domains/moonzdevelopment.json:7]: That's just the beginning to create my profile

File: domains/hymoonz.json (hunk 1 above)
Comment (hymoonz) [reply domains/hymoonz.json:7]: That's just the beginning to create my profile

Comment (hymoonz) [reply domains/hymoonz.json:7]: No sir

Comment (hymoonz) [reply domains/hymoonz.json:7]: I've changed it

File: domains/moonzdevelopment.json (hunk 2 above)
Comment (hymoonz) [reply domains/moonzdevelopment.json]: I've changed it

**Pull request closed without merging**